core/renderers/common/i_path_object.js

```diff
@@ -15,15 +15,15 @@
 goog.module('Blockly.blockRendering.IPathObject');
 goog.module.declareLegacyNamespace();
 
-goog.requireType('Blockly.Block');
-goog.requireType('Blockly.blockRendering.ConstantProvider');
-goog.requireType('Blockly.Theme');
+const Block = goog.requireType('Blockly.Block');
```

Comment:
Do these need comments to silence the linter?

Reply:
yes, looks like I forgot to add them. I'll add them now.